staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -472,6 +472,9 @@ type PersistentVolumeClaimSpec struct {
 	// +optional
 	Selector *metav1.LabelSelector `json:"selector,omitempty" protobuf:"bytes,4,opt,name=selector"`
 	// Resources represents the minimum resources the volume should have.
+	// If RecoverVolumeExpansionFailure is enabled users are allowed to specify resource requirements
```

Comment:
unclear what `If RecoverVolumeExpansionFailure is enabled` means - can you say something like "If the RecoverVolumeExpansionFailure feature gate is enabled..."

Reply:
fixed.